random-generator.install

```diff
@@ -0,0 +1,19 @@
+lib: [
+  "_build/install/default/lib/random-generator/META" {"META"}
```

Comment:
just a note: install files should not normally be in git. They're generated by dune when the library is built.

Reply:
Thanks! I forgot to .gitignore this :)